src/test/java/com/spring/AddressServiceTest.java

```diff
@@ -0,0 +1,56 @@
+package com.spring;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import com.spring.address.db.models.Address;
+import com.spring.address.db.repositories.AddressRepository;
+import com.spring.address.service.AddressService;
+
+@ExtendWith(MockitoExtension.class)
+public class AddressServiceTest {
+	
+	@InjectMocks
+	private AddressService addressService;
+	
+	@Mock 
+	private AddressRepository addressRepository;
+	
+	private Address address = Address.builder().id(1L).town("Khm").street("Prospect").houseNumber("34").build();
```

Comment:
Do not write builder calls in one single line. Rewrite here and everywhere in the project as follows:
```
Address.builder()
    .id(1L)
    .town("Khm")
    .street("Prospect")
    .houseNumber("34")
    .build();
```
Also consider making the field `final` as it's never going to change.